test/Microsoft.ML.OnnxTransformTest/DnnImageFeaturizerTest.cs

```diff
@@ -61,9 +61,11 @@ public DnnImageFeaturizerTests(ITestOutputHelper helper) : base(helper)
         [ConditionalFact(typeof(Environment), nameof(Environment.Is64BitProcess))]
         void TestDnnImageFeaturizer()
         {
+            // Onnxruntime supports Ubuntu 16.04, but not CentOS
```

Comment:
Can you use the `[OnnxFact]` here instead? And remove the

```
            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
                return;
```

below?

Reply:
Leaving it as-is for now. Switching to OnnxFact is causing some other odd issues.